eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -1,6 +1,6 @@
 steps:
   - pwsh: |
-      Write-Host '##vso[task.setvariable variable=VCPKG_BINARY_SOURCES_SECRET;issecret=true;]clear;x-azblob,https://azuresdkartifacts.blob.core.windows.net/public-vcpkg-container,,read'
+      Write-Host '##vso[task.setvariable variable=VCPKG_BINARY_SOURCES_SECRET;issecret=true;]clear;x-azcopy,https://azuresdkartifacts.blob.core.windows.net/public-vcpkg-container,read'
```

Comment:
https://github.com/Azure/azure-sdk-for-rust/blob/main/eng/scripts/Set-VcpkgWriteModeCache.ps1#L45

I assume you need to update that as well

Reply:
Not required at this point because only `x-azblob` without an SAS behavior was changed... but it's the right thing to update.